src/proc/colorizer.cpp

```diff
@@ -245,7 +245,8 @@ namespace librealsense
         if (!frame || frame.is<rs2::frameset>())
             return false;
 
-        if (frame.get_profile().stream_type() != RS2_STREAM_DEPTH)
+        if ((frame.get_profile().stream_type() != RS2_STREAM_DEPTH) || 
+            (frame.get_profile().format() != RS2_FORMAT_DISPARITY32) && (frame.get_profile().format() != RS2_FORMAT_Z16))
```

Comment:
Why `RS2_FORMAT_DISPARITY32` ?

Reply:
and why NOT Z16?

Reply:
If the depth format is `not Z16` & `not DISPARITY32`, then `return FALSE`.

Reply:
I understand, 
Maybe it will be more readable if we will add some static array of supported formats and check here if the frame format is in the supported formats?